ajax/libs/inferno-component/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "inferno-component",
+  "version": "3.6.4",
+  "filename": "inferno-component.min.js",
+  "license": "MIT",
+  "description": "Provides ES2015 stateful components (with lifecycle events) to Inferno",
+  "author": {
+    "name": "Dominic Gannaway",
+    "email": "dg@domgan.com"
+  },
+  "homepage": "https://infernojs.org/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/infernojs/inferno.git"
+  },
+  "keywords": [
+    "babel",
+    "react",
+    "inferno",
+    "framework",
+    "interfaces",
+    "user interfaces",
+    "html",
+    "renderToString",
+    "server",
+    "dom",
+    "browser",
+    "rollup"
+  ],
+  "npmName": "inferno-component",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/inferno-component!(*.node|*.es2015)*.+(js|css|map)"
```

Comment:
`*.es2015`: v0.5.20~v0.5.22, v0.6.0~0.6.4, v0.7.2, v0.7.3
`node`: 1.3.0~1.4.0
Before I manually add v0.5.12 ~ v1.2.2, `*.es2015` mach v0.5.20~v0.5.22, v0.6.0~0.6.4, v0.7.2, v0.7.3. but now because I manually add v0.5.12 ~ v1.2.2, I'll remove this pattern.